index.html

```diff
@@ -1,74 +1,109 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
```

Comment:
You missed the viewport meta tag, and your media queries don't work due to that

Reply:
Viewport meta tag added